Content/RELEASE_NOTES.md

```diff
@@ -0,0 +1,2 @@
+### 0.0.1 - 1970-01-01
```

Comment:
can we somehow fill that date by template ?

Reply:
yes it's handled by the "createdate" symbol in template.json